app/assets/stylesheets/layouts/_header_navbar.scss

```diff
@@ -1,13 +1,92 @@
+$bg-default      : #0088cc;
+$bg-highlight    : #23527c;
+$col-default     : #eeeeee;
+$col-highlight   : #ffffff;
+$dropdown       : false;
+
 body {
-  padding-top: $header-navbar-height;
+  padding-top: $navbar-height;
 }
 
-// application nav bar
+.navbar-default {
+  background-color: $bg-default;
+  border-color: transparent;
+
+  .navbar-brand {
+    color: $col-default;
+
+    &:hover, &:focus {
+      color: $col-highlight;
+    }
+  }
+
+  .navbar-text {
+    color: $col-default;
+  }
+
+  .navbar-nav {
+    > li {
+      > a {
+        color: $col-default;
+
+        &:hover,  &:focus {
+          color: $col-highlight;
+        }
+      }
+
+      @if $dropdown {
+        > .dropdown-menu {
+          background-color: $bg-default;
+          > li > a {
+            color: $col-default;
+            &:hover,  &:focus {
+              color: $col-highlight;
+              background-color: $bg-highlight;
+            }
+          }
+          .divider {
+            background-color: $bg-highlight;
+          }
+        }
+      }
+    }
+    > .active {
+      > a, > a:hover, > a:focus {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Each selector in a comma sequence should be on its own line